components/brave_ads/core/internal/account/utility/redeem_confirmation/redeem_confirmation_feature.h

```diff
@@ -0,0 +1,27 @@
+/* Copyright (c) 2024 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+#ifndef BRAVE_COMPONENTS_BRAVE_ADS_CORE_INTERNAL_ACCOUNT_UTILITY_REDEEM_CONFIRMATION_REDEEM_CONFIRMATION_FEATURE_H_
+#define BRAVE_COMPONENTS_BRAVE_ADS_CORE_INTERNAL_ACCOUNT_UTILITY_REDEEM_CONFIRMATION_REDEEM_CONFIRMATION_FEATURE_H_
+
+#include "base/feature_list.h"
+#include "base/metrics/field_trial_params.h"
+#include "base/time/time.h"
+
+namespace base {
+class TimeDelta;
+}  // namespace base
+
+namespace brave_ads {
+
+BASE_DECLARE_FEATURE(kRedeemConfirmationFeature);
+
+inline constexpr base::FeatureParam<base::TimeDelta> kFetchPaymentTokenAfter{
```

Comment:
This feature is only for rewards users. This file and feature should be renamed to match`redeem_reward_confirmation`.